src/track/TrackObject.ts

```diff
@@ -315,6 +317,10 @@ export class TrackObject<
         this.triggerDisplayUpdate();
     }
 
+    protected onLoadFailed = () => {
+        this.loadingIndicator.string = 'Track could not be loaded';
+        this.loadingIndicator.color = [1, 0, 0, 1];
+    }
```

Comment:
Is this error message the same font and font-weight as the other error messages? It was hard for me to tell, but I think it might not be.

Reply:
The color is red. The way it works is that the color array is [R, G, B, alpha] where 
'R' is red, 'G' is green and 'B' is blue. Alpha is the intensity. So, the color is red.

Reply:
The color is good! I wanted to check about the font and font-weight.